libs/core/src/components/pds-tooltip/pds-tooltip.tsx

```diff
@@ -97,6 +97,19 @@ export class PdsTooltip {
     this.el.addEventListener('focus', this.handleShow, true);
   }
 
+  componentDidLoad() {
+    // fix for Safari iOS back button issue
+    const handlePageShow = () => {
+      this.opened = false;
+    };
```

Comment:
Does this need to live here? Could this function be moved outside of the `componentDidLoad`

Reply:
yup. I moved it, let me know if that's what you had in mind.